README.md

```diff
@@ -4,7 +4,11 @@
 
 # Set up and mobile skeleton
 
-> This project is the first part of my portfolio website. More will be added later.
+> In this project, I deploy my portfolio website.
+
```

Comment:
- [ ] Kindly add a  link of the deployed website to your readme file, for example: 
```
## live demo
[live demo](https://dd-obua.github.io/Dan-portfolio/)
```

So your link looks like this: [live demo](https://dd-obua.github.io/Dan-portfolio/)